M/MFEM/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MFEM_seq"
```

Comment:
Note: this name must match exactly the name of the directory, so either change this name or move the directory.

Reply:
The idea was to compile MPI MFEM later (it's harder)
Can I use the same structure as MUMPS, that is move this build_tarballs.jl in an inner folder like
M / MFEM / MFEM_seq / build_tarballs.jl
?

Reply:
That's OK, yes